py/examples/query_file.json

```diff
@@ -0,0 +1,9 @@
+[
+  {
+    "query": "avg:redis.mem.peak{app_feature:shared} by {shared_resource_id}.rollup(5)",
+    "unit": "bytes"
+  }, {
+    "query": "avg:redis.cpu.user{app_feature:shared} by {shared_resource_id}",
+    "unit": "milliseconds"
```

Comment:
This will be seen as an example, so we should show an example of a suitable metric for the datadog fetcher.
avg(redis.mem.peak) and avg(redi.cpu.user) are not good examples because:
- These are system metrics, they will never have an `app_feature` tag. if redis was able to record the breakdown by app_feature we would not need this work.
- They are averages. We will want sums and rollup if counters as people will be doing accounting of how many bytes, how many seconds the app_feature is consuming. 
- - Ideally the metric should be a counter.
- - the aggregation should be a sum
- - then you can add a rollup if you want but not required.

So please find a more representative query.